src/AdvApi32.Tests/AdvApi32.cs

```diff
@@ -1,10 +1,7 @@
 ﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 
-using System;
-using PInvoke;
 using Xunit;
-using static PInvoke.AdvApi32;
```

Comment:
I did the removal in VS on solution scope via VS analyzerd but can undo for all non-splitted files